metadata-transformer/src/test/java/no/unit/nva/metadata/service/MetadataServiceTest.java

```diff
@@ -250,23 +262,23 @@ private static CreatePublicationRequest createRequestWithTags(List<String> tags)
         CreatePublicationRequest emptyRequest = createRequestWithIdentifier(null, dummyTitle);
 
         return Stream.of(
-                generateMetadataHtml(Map.of(
-                        DC_TITLE, dummyTitle,
-                        DC_IDENTIFIER, doiWithPrefix),
-                        request),
-                generateMetadataHtml(Map.of(
-                        DC_TITLE, dummyTitle,
-                        DC_IDENTIFIER_UPPER_CASE, doiWithoutPrefix),
-                        request),
-                generateMetadataHtml(Map.of(
-                        DC_TITLE, dummyTitle,
-                        DC_IDENTIFIER_UPPER_CASE, expectedDoi),
-                        request),
-                generateMetadataHtml(Map.of(
-                        DC_TITLE, dummyTitle,
-                        DC_IDENTIFIER, notADoi),
-                        emptyRequest)
-                );
+            generateTestWithMetadataHtml("dc.identifier with doi prefix map to request", Map.of(
```

Comment:
…request...DOI?

Reply:
I was thinking more.. "map to createRequest" maybe "to doi in createRequest" 
am I off base?

Reply:
doi in createRequest, I think